python/paddle/incubate/optimizer/lbfgs.py

```diff
@@ -118,18 +124,18 @@ class LBFGS(Optimizer):
 
     def __init__(
         self,
-        learning_rate=1.0,
-        max_iter=20,
-        max_eval=None,
-        tolerance_grad=1e-7,
-        tolerance_change=1e-9,
-        history_size=100,
-        line_search_fn=None,
-        parameters=None,
-        weight_decay=None,
-        grad_clip=None,
-        name=None,
-    ):
+        learning_rate: float = 1.0,
+        max_iter: int = 20,
+        max_eval: int | None = None,
+        tolerance_grad: float = 1e-7,
+        tolerance_change: float = 1e-9,
+        history_size: int = 100,
+        line_search_fn: str | None = None,
+        parameters: list[Tensor] | tuple[Tensor] | None = None,
```

Comment:
```suggestion
        parameters: Sequence[Tensor] | Sequence[_ParameterConfig] | None = None,
```

参考 `Optimizer` 的初始化参数